pages/community-call.jsx

```diff
@@ -58,8 +58,11 @@ var CommunityCallPage = React.createClass({
 
           <ul className="upcoming-workshops">
             <li>
-              <p className="date">October TBD</p>
-              <h2>Theme: TBD</h2>
+              <p className="date">October 13, 2016</p>
+              <h2>Ada Lovelace Day</h2>
+              <p>
+                Join the Mozilla Learning and Mozilla Science teams to celebrate Ada Lovelace Day and women around the globe who are making a difference in science, web literacy and beyond.
```

Comment:
As an aside, here's our [post](https://science.mozilla.org/programs/events/community-call-oct-13-11-et) feel free to cross-link! Would love to link to yours as well!